### ISSUE TITLE
Issue generating example shinylive output on windows 10 

### ISSUE THREAD

**Issue by Dariom92** [quarto-ext/shinylive#3]
I am unable to get the example code shown on the readme to correctly render as a shiny app on windows 10.  I created a quarto project with the below  files an ran `quarto render` and got  the following output: 

<pre>pandoc
  to: html
  output-file: ShinyQuarto.html
  standalone: true
  section-divs: true
  html-math-method: mathjax
  wrap: none
  default-image-extension: png

metadata
  document-css: false
  link-citations: true
  date-format: long
  lang: en
  title: Quarto Basics

WARNING: Specified QUARTO_DENO does not exist, using built in deno
Output created: ShinyQuarto.html </pre>

The rendered .html just shoes the code in the shinylive python cell rather than showing the app. 

Quartro version: 1.2.247
python version:  3.9.13

## ShinyQuarto.qmd
<pre>```
---
title: Shinylive in Quarto example
format: html
filters:
  - shinylive
---

This is a Shinylive application embedded in a Quarto doc.

```{shinylive-python}
#| standalone: true

from shiny import *

app_ui = ui.page_fluid(
    ui.input_slider("n", "N", 0, 100, 40),
    ui.output_text_verbatim("txt"),
)

def server(input, output, session):
    @output
    @render.text
    def txt():
        return f"The value of n*2 is {input.n() * 2}"

app = App(app_ui, server)
```</pre>

# _quarto.yml
<pre>project:
  title: "ShinyQuarto"
filters:
  - shinylive
</pre>

**Comment by wch** [comment]
If you open up the Javascript console in your browser, what does it say?

Also, what files are in the `docs/site_libs/quarto-contrib/` subdirectory? There should be `shinylive-0.0.11` and `shinylive-quarto-css`.

**Comment by Dariom92** [comment]
> If you open up the Javascript console in your browser, what does it say?
> 
> Also, what files are in the `docs/site_libs/quarto-contrib/` subdirectory? There should be `shinylive-0.0.11` and `shinylive-quarto-css`.

You are correct there were some errors in the javascript console please see below. 

I am not sure what "docs" folder are you referring to sorry. The extension is in the same folder as the quarto project under `_extensions` if that's what you mean. 

![JavascriptErrors](https://user-images.githubusercontent.com/8896743/198847538-fedcdbfb-8455-4fa5-80d5-f2145ed40a49.PNG)

**Comment by wch** [comment]
I think the problem is that you're trying to open the file directly (via a `file:///` URL) instead of through a web server (via `http://` or `https://` URL). You can use `quarto preview` to load it with a web server.

For security reasons, more and more modern JS and WASM code needs to be loaded through a web server.
